examples/advanced/quantum_volume_errors.ipynb

```diff
@@ -71,7 +71,7 @@
         "    num_circuits=num_circuits,\n",
         "    depth=depth,\n",
         "    num_qubits=depth,\n",
-        "    device_graph=routing.gridqubits_to_graph_device(device.qubits),\n",
+        "    device_graph=routing.gridqubits_to_graph_device(device.metadata.qubit_set),\n",
```

Comment:
Optional:  I wonder if it would be worth having a deprecated qubits() function in GridDevice to point to metadata.qubit_set?  This would help people know how to upgrade this everywhere.  This would be a separate PR of course.  Just an idea.

Reply:
Good idea, will do!